screens/Home/index.js

```diff
@@ -1,18 +1,11 @@
-import React, { useEffect, useContext } from 'react';
+import React from 'react';
 import { View, StyleSheet } from 'react-native';
 import { Text, Card, ListItem } from 'react-native-elements';
 
-import { AppContext } from 'context';
 import { TRIVIA_SCREEN } from 'screens/routes';
 import colors from 'consts/colors';
 
 const Home = ({ navigation }) => {
-  const { getAllCategoryData } = useContext(AppContext);
-
-  useEffect(() => {
-    getAllCategoryData();
```

Comment:
Moved this to the Trivia screen, was finding that Context was causing this to fire when other functions were fired in that AppContext. Specifically, after selecting a difficulty it then calls getTriviaData in AppContext - I noticed in the console logs that the home page was re-rendering or something. Was hard to tell why it wanted to, the entire app is wrapped in this context so I guess we need to be mindful about this going forward. Moving it to the Trivia screen has worked well though, and is firing at the appropriate times.